server/src/controllers/contactController.ts

```diff
@@ -243,6 +243,176 @@ contactController.get('/:id', async (req: Request, res: Response) => {
   }
 });
 
+/**
+ * @swagger
+ * /2/contact/orders/{id}:
+ *   get:
+ *     summary: get a contact including orders and donations
+ *     tags:
+ *     - New Contact
+ *     parameters:
+ *     - $ref: '#/components/parameters/id'
+ *     security:
+ *      - bearerAuth: []
+ *     responses:
+ *       200:
+ *         description: Contact got successfully.
+ *       400:
+ *         description: bad request
+ *         content:
+ *           application/json:
+ *             schema:
+ *               $ref: '#/components/schemas/Error'
+ *       500:
+ *         description: Internal Server Error. An error occurred while processing the request.
+ */
+contactController.get('/orders/:id', async (req: Request, res: Response) => {
+  try {
+    const id = req.params.id;
+    const contact = await prisma.contacts.findUnique({
+      where: {
+        contactid: Number(id),
+      },
+      include: {
+        orders: {
+          orderBy: [{
+            orderdate: 'desc',
+          }, {
+            ordertime: 'desc',
+          }],
+          select: {
+            orderid: true,
+            orderdate: true,
+            ordertime: true,
+            refund_intent: true,
+            ordertotal: true,
+            orderitems: {
+              select: {
+                price: true,
+                singletickets: {
+                  select: {
+                    ticketwasswapped: true,
+                    eventtickets: {
+                      select: {
+                        ticketrestrictions: {
+                          select: {
+                            tickettype: {
+                              select: {
+                                description: true,
+                              },
+                            },
+                          },
+                        },
+                        eventinstances: {
+                          select: {
+                            eventdate: true,
+                            eventtime: true,
+                            events: {
+                              select: {
+                                eventname: true,
+                              },
```

Comment:
It would be great it the `orderitems` view included the season that the event is in, if any. That might be too much of a lift for this PR though? It's not a blocker for me if you want to move that into another ticket

Reply:
This should be pretty easy

Reply:
<img width="1371" alt="image" src="https://github.com/WonderTix/WonderTix/assets/92124260/6e3a1b1f-38ce-4243-a823-f0d4f7f95bfe">
(event not in season and event in season)